evergreen/config.yml

```diff
@@ -144,6 +144,18 @@ functions:
             -C cmake_vars.txt ${extra_flags} \
             -G "$GENERATOR"
 
+          # Fail the test if local object store tests are defined and this is not a patch build
+          if [[ -f "build/objstore-tests.do-not-commit" ]]; then
+              if [[ -z "${is_patch}" ]]; then
+                echo "ERROR: Found custom object store tests for non-patch build"
+                echo "- Revert test/object-store/objstore-tests.do-not-commit and recommit"
+                exit 1
+              else
+                echo "WARNING: Custom object store tests are configured for this patch build"
+                echo "- Revert test/object-store/objstore-tests.do-not-commit before merging PR"
```

Comment:
maybe a better approach would be to allow the tests to run, but then fail the patch anyways to make sure this warning is not missed. otherwise it will probably get lost in a sea of test output as written.

Reply:
Sure - I'll see about moving this check to the end of the tests run so it at least shows up on log tab of the evergreen task page.